burn-candle/src/ops/int_tensor.rs

```diff
@@ -311,6 +311,14 @@ impl<F: FloatCandleElement, I: IntCandleElement> IntTensorOps<Self> for Candle<F
         CandleTensor::new(tensor.tensor.sum_keepdim(dim).unwrap())
     }
 
+    fn int_prod<const D: usize>(tensor: IntTensor<Self, D>) -> IntTensor<Self, 1> {
+        todo!();
```

Comment:
tch, ndarray & candle dont support prod_axis required for this implementation. 

QUESTION should I implement it in the burn layer with some gymnastics or leave it as panic!() like e.g. burn-candle's int_div_scalar

Reply:
I think it's rather bad to offer an operation in the API if it's gonna fail on 75% of our backends. I think if you can work out some default implementation in the burn layer it would be cool, otherwise we should not even offer the operation. 
Candle's int_div_scalar is an exception that we should fix eventually.

Reply:
i agree. i will file issues on tch, ndarray and candle and implement a default.